shell/platform/darwin/ios/framework/Source/FlutterViewController.mm

```diff
@@ -1799,11 +1857,74 @@ - (void)scrollEvent:(UIPanGestureRecognizer*)recognizer API_AVAILABLE(ios(13.4))
     _mouseState.last_translation = CGPointZero;
   }
 
+  if (_mouseState.flutter_state_is_added) {
+    packet->SetPointerData(/*index=*/0, pointer_data);
+  } else {
+    flutter::PointerData add_pointer_data = pointer_data;
+    add_pointer_data.signal_kind = flutter::PointerData::SignalKind::kNone;
+    add_pointer_data.change = flutter::PointerData::Change::kAdd;
+    _mouseState.flutter_state_is_added = true;
+    packet->SetPointerData(/*index=*/0, add_pointer_data);
+    packet->SetPointerData(/*index=*/1, pointer_data);
```

Comment:
I assume this is manually adding a "add" event. Is this something we normally do? Is there a side effect for this?
@chunhtai might know.

Reply:
this looks correct